packages/react-core/src/components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -55,12 +55,10 @@ export class OverflowMenu extends React.Component<OverflowMenuProps, OverflowMen
   };
 
   render() {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { className, breakpoint, children, ...props } = this.props;
     return (
-      <div
-        {...props}
-        className={css(styles.overflowMenu, getModifier(styles.modifiers, `showOn ${breakpoint}`), className)}
-      >
+      <div {...props} className={css(styles.overflowMenu, className)}>
```

Comment:
breakpoint class `showOn..` missing

Reply:
Based on the implementation of `getModifier` since there was a space like `showOn ${breakpoint}` rather than `showOn${breakpoint}` the class was never getting added. I'm preserving this (maybe broken?) behavior for now.

Reply:
Double checked with core styles, and overflow menu does NOT have breakpoint modifiers. It has it's own JS implementation for the breakpoints in `handleResize`, so this is safe to remove.